server/models/ride.ts

```diff
@@ -15,6 +15,8 @@ export enum Status {
   ARRIVED = 'arrived',
   PICKED_UP = 'picked_up',
   COMPLETED = 'completed',
+  NO_SHOW = 'no_show',
+  LATE = 'late',
```

Comment:
Might want to keep track of lateness separately. Because then, once the ride is complete, we will lose the data indicating that the person had been late, which we might want to know later on.